src/libs/antares/study/runtime/runtime.cpp

```diff
@@ -485,6 +486,23 @@ void StudyRuntimeInfos::checkThermalTSGeneration(Study& study)
     });
 }
 
+static void checkMiscGenUsed(const AreaList& areas)
```

Comment:
- We have this kind of log message : 
![log-misc-gen](https://user-images.githubusercontent.com/62292552/164421867-8e66a03e-683d-4067-a16a-ef6192cbc82b.PNG)
so there is a log line for each area name.
For more tightness, we should print a list of area names in a row.
- Why not print a warning instead of a classic log ?
As is, users do not know there should use renewable clusters instead of misc. Gen.

Reply:
- OK
- When loading the study, unless the user asks to ignore warnings, all warnings and errors are fatal. I'm not sure we'd want to go as far as this.

Reply:
Regarding second bullet, it seems that warnings do not disable stages optimization and results print.
I just made the test.
Besides, ticking option **ignore warnings** in the **Run the simulation** window do not seem to have an effect : warnings are still shown in a separate windows at end of simulation.

Reply:
Done. All in one line + warning